image/scripts/install_dependencies.sh

```diff
@@ -66,12 +65,38 @@ if [ "$PKGSTOINSTALL" != "" ]; then
         echo "" 1>&2
         exit -1
     fi
+
+    # upgrade
+    sudo apt-get -y upgrade | tee /tmp/bibos_upgrade_log.txt
+    sudo apt-get -y dist-upgrade | tee /tmp/bibos_upgrade_log.txt
+
     # Clean .deb cache to save space
-    sudo apt-get clean
+    sudo apt-get -y autoremove
+    sudo apt-get -y clean
 
 fi
 
 # Install python packages
-sudo pip install bibos-client
+pip install --upgrade bibos-utils bibos-client
+
+# Enable security updates to be automatically installed
+CONF="/etc/apt/apt.conf.d/90os2borgerpc-automatic-upgrades"
+
+if ! dpkg -s unattended-upgrades > /dev/null 2>&1; then
+    sudo apt-get -y install unattended-upgrades
+fi
+    cat > "$CONF" <<END
```

Comment:
This `cat` will fail if we're not running as `root` (since we're trying to write to a system configuration folder). But, if we're running as root, why are there so many `sudo` invocations in this file?

(See, however, the next comment.)

Reply:
In the documentation (https://github.com/OS2borgerPC/image/blob/master/doc/HOWTOCreate_a_new_BibOS_image_from_scratch.txt) the scripts is not executed as root... But that could be changed...